main.cpp

```diff
@@ -150,47 +147,45 @@ int main()
 				update_letters_used(chars_already_guessed, pos);
 				fill_letter(word_copy, hangman, pos);
 
-				display_next_combo_name(s_tries, s_combos, combo_length);
-				s_tries++;
+				display_next_combo_name(correct_letters_guessed, success_combo);
+				correct_letters_guessed++;
 
-				// check for duplicates
-				int pos2 = lowercased.find(usr_input, pos + 1);
+				size_t duplicate = lowercased.find(user_input, pos + 1);
 
-				// check if duplicate was found
-				if (pos2 != -1)
+				if (duplicate != std::string::npos)
 				{
-					update_letters_used(chars_already_guessed, pos2);
-					fill_letter(word_copy, hangman, pos2);
+					update_letters_used(chars_already_guessed, duplicate);
+					fill_letter(word_copy, hangman, duplicate);
 
-					display_next_combo_name(s_tries, s_combos, combo_length);
-					s_tries++;
+					display_next_combo_name(correct_letters_guessed, success_combo);
+					correct_letters_guessed++;
 				}
 			}
 			else
 			{
 				std::cout << "You've guessed that already, try again\n\n";
-				f_tries++;
+				failed_attempts++;
 			}
 		}
-		else if (isdigit(usr_input))
+		else if (isdigit(user_input))
 		{
-			display_next_combo_name(f_tries, f_combos, combo_length);
-			f_tries++;
+			display_next_combo_name(failed_attempts, failed_combos);
+			failed_attempts++;
 		}
 		else
 		{
-			display_next_combo_name(f_tries, f_combos, combo_length);
-			f_tries++;
+			display_next_combo_name(failed_attempts, failed_combos);
+			failed_attempts++;
 		}
 
-		if (s_tries == len_of_word)
+		if (correct_letters_guessed == len_of_word)
 		{
 			std::cout << word << " is the word, You WIN!!\n";
 			break;
 		}
 	}
 
-	if (f_tries == tries)
+	if (failed_attempts == tries)
```

Comment:
Prefer to always add braces even for single line blocks, else they can cause unexpected issues after a merge. Eg:
```cpp
// 1
if (foo)
    bar = 1;

// 2
if (foo)
    bar = 0;

// Merged
if (foo)
    bar = 1;
    bar = 0;    // not under if anymore!
```